### DISCUSSION TITLE
git storage won't sync

### DISCUSSION THREAD

**Discussion by bwoodill** [requarks/wiki#5898] [Error / Bug Report]
I have tried connecting to github, gitlab, and self hosted gitlab and I cannot get the git backend to work. I have tried using ssh and https, using a .pem file and pasting the key into the field, verify ssl on/off, made sure email/name matched and different options, different local repository options, bi-direction or just pull, and nothing. Also when I tried the key file and local repo options I made sure to chmod 777 to make sure no permissions issues were going on on the file system. 

No matter what I have tried the error is always:
Error: spawn git ENOENT at Process.ChildProcess._handle.onexit (node:internal/child_process:285:19) at onErrorNT (node:internal/child_process:485:16) at processTicksAndRejections (node:internal/process/task_queues:83:21)

Any suggestions? Host is deb11

**Comment by NGPixel** [comment]
Is git installed on your system and accessible?
